packages/app/src/cli/commands/app/generate/extension.ts

```diff
@@ -67,7 +67,7 @@ export default class AppScaffoldExtension extends Command {
   static args = [{name: 'file'}]
 
   public static analyticsNameOverride(): string | undefined {
-    return 'scaffold'
+    return 'dev scaffold extension'
```

Comment:
I think this is wrong

```suggestion
    return 'app scaffold extension'
```